File: frontend/tcc/src/app/models/dto/vereador-dto.ts
export interface VereadorDTO {
  id: number;
  nome: string;
  partido: string;
  legislaturas: string;
  gabinete: string;
  telefone: string;
  email: string;
  site: string;
  genero: string;
  vereador_ativo: string;
  dataNasc: string;
  vereador_cor: string;
  ocupacao: string;
  vereador_escolaridade: string;
  avatarUrl: string;
  seguidores?: number;
}

File: backend/clientes_TCC/src/main/java/com/clientes/clientes_TCC/domain/Vereador/VereadorDTO.java
package com.clientes.clientes_TCC.domain.Vereador;

import com.clientes.clientes_TCC.domain.Partido.Partido;

import java.util.Date;

public record VereadorDTO(
        Integer id,
        String nome,
        Partido partido,
        String email,
        String legislaturas,
        String gabinete,
        String telefone,
        String site,
        VereadorAtivo vereador_ativo,
        String genero,
        Date dataNasc,
        VereadorCor vereador_cor,
        String ocupacao,
        VereadorEscolaridade vereador_escolaridade
) {
}
